app/main/forms/suppliers.py

```diff
@@ -96,6 +96,7 @@ class AddCompanyRegistrationNumberForm(FlaskForm):
     )
     companies_house_number = DMStripWhitespaceStringField(
         'Companies House number',
+        id='companies_house_number',
```

Comment:
Because these are nested the ids have the id of the component they are nested under prepended by the frontend toolkit template, so this should be

```suggestion
        id='input-has_companies_house_number-1-companies_house_number',
```

Or at least that what the id for the input is in Chrome dev tools on my machine